ts/private/ts_project.bzl

```diff
@@ -218,7 +218,7 @@ This is an error because Bazel does not run actions unless their outputs are nee
     if len(outputs) > 0:
         inputs_depset = depset(
             copy_files_to_bin_actions(ctx, inputs),
-            transitive = transitive_inputs + [_gather_declarations_from_js_providers(ctx.attr.srcs + ctx.attr.deps)],
+            transitive = transitive_inputs + [_gather_declarations_from_js_providers(ctx.attr.srcs + [ctx.attr.tsconfig] + ctx.attr.deps)],
```

Comment:
Isn't it possible there will be no `tsconfig` so `[None]` will be passed to `_gather_declarations_from_js_providers`? Is that ok?

Reply:
I checked the same thing. Not possible as the attribute is `mandatory = True` in the rule. You either get the user tsconfig or an auto-generated one that is done in the `ts_project` macro

Reply:
👍